src/test/ui/cfg/cfg-target-compact.rs

```diff
@@ -0,0 +1,10 @@
+// run-pass
+#![feature(cfg_target_compact)]
+
+#[cfg(target(os = "linux", pointer(width = "64")))]
```

Comment:
What is the `pointer(width = "64")` syntax?
The RFC doesn't specify it and it complicates the implementation.

Reply:
Same as `target(..)`, `pointer(width = "64")` -> `pointer_width = "64"`.
But as requested I'm removing this.